### DISCUSSION TITLE
Adding IP address to host file on windows image

### DISCUSSION THREAD

**Discussion by mrking** [community/community#25959] [Actions]
<p>I’ve been trying to figure out how to map a DNS name to an unlisted publicly hosted server for running integrated SDK tests against. Instead of getting the desired result, I get some strange result with a URL that seems to be somehow related bitcoin mining when searched on google.</p>
<p>The log goes something like this and I can’t quite figure out what’s going on. The fact the IP address shows up appended to another URL when attempting to ping is strange. Initially I thought it required a space or line feed but that doesn’t appear to help.</p>
<p>Add test server host name</p>
<p>0s</p>
<p>Run ECHO %NEWLINE%^1.2.3.4 <a href="http://example.org" rel="nofollow noopener">example.org</a>&gt;&gt;%WINDIR%\System32\drivers\etc\hosts</p>
<p>debug dns name to make sure ip is assigned</p>
<p>3s</p>
<p>Run ping <a href="http://example.org" rel="nofollow noopener">example.org</a></p>
<p>Pinging stratum-na.rplant.xyz1.2.3.4 [127.0.0.1] with 32 bytes of data:</p>
<p>Reply from 127.0.0.1: bytes=32 time&lt;1ms TTL=128</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/mrking">@mrking</a>,</p>
<p>Glad to see you in Github Community Forum!</p>
<p>This is due to <code>1.2.3.4 example.org</code> is added behind <code>stratum-na.rplant.xyz</code>  while not in a new line.<br>
The default shell for windows hosted runner is pwsh, so please use below command instead:</p>
<pre><code class="lang-auto">echo "`n1.2.3.4 example.org"&gt;&gt;C:\Windows\System32\drivers\etc\hosts   #add to new line
</code></pre>
<p>Please check my workflow for your refernce.<br>
</p><aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/weide-zhou/ticket13/runs/871845370?check_suite_focus=true#step:5:185" target="_blank" rel="nofollow noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/52772761/181094306-b7d5fb11-4a79-4a5d-b9ba-d81f02202427.png" class="thumbnail onebox-avatar" width="420" height="420">

<h3><a href="https://github.com/weide-zhou/ticket13/runs/871845370?check_suite_focus=true#step:5:185" target="_blank" rel="nofollow noopener">weide-zhou/ticket13</a></h3>

<p>Contribute to weide-zhou/ticket13 development by creating an account on GitHub.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Thanks</p>

**Comment by mrking** [comment]
<p><a class="mention" href="/weide-zhou">@weide-zhou</a></p>
<p>Thanks so much! I had been using CMD, but this seems to have worked! Really appreciate taking the time to even setup a working example.</p>
